extensions/shared/external-media/editor.scss

```diff
@@ -0,0 +1,363 @@
+/**
+ * External Media
+ */
+@import '../styles/gutenberg-base-styles.scss';
+
+$grid-size: 8px;
+
+@keyframes loading-fade {
+	0% {
+		opacity: 0.5;
+	}
+	50% {
+		opacity: 1;
+	}
+	100% {
+		opacity: 0.5;
+	}
+}
+
+/**
+ * Media item container
+ */
+
+.jetpack-external-media-browser__is-copying {
+	max-height: 20%;
+	max-width: 20%;
+	min-width: 480px;
+	min-height: 360px + 56px;
+
+	.jetpack-external-media-browser__single {
+		height: 300px;
+	}
+
+	.jetpack-external-media-browser__media {
+		height: 100%;
+	}
+
+	.is-transient {
+		border: 0;
+		background: 0;
+		padding: 0;
+		height: 100%;
+
+		&.jetpack-external-media-browser__media__item__selected {
+			box-shadow: none;
+			border-radius: 0;
+		}
+
+		img {
+			width: 100%;
+			height: 100%;
+			object-fit: contain;
+		}
+	}
+}
+
+.jetpack-external-media-browser:not( .jetpack-external-media-browser__is-copying ) {
+	.is-error {
+		margin-bottom: 1em;
+		margin-left: 0;
+		margin-right: 0;
+	}
+
+	.components-placeholder {
+		background-color: transparent;
+	}
+
+	.components-modal__content {
+		overflow: auto;
+		padding-bottom: 0;
+	}
+}
+
+@media ( min-width: 600px ) {
+	.components-modal__content {
```

Comment:
Needs prefixing: https://github.com/Automattic/jetpack/issues/16143#issuecomment-642857130

Reply:
Fixed in #16144